script/validate-gofmt

```diff
@@ -3,7 +3,7 @@
 source "$(dirname "$BASH_SOURCE")/.validate"
 
 IFS=$'\n'
-files=( $(validate_diff --diff-filter=ACMR --name-only -- '*.go' | grep -v '^\(integration/\)\?vendor/' || true) )
+files=( $(validate_diff --diff-filter=ACMR --name-only -- '*.go' | grep -v '^\(integration/\)\?vendor/\|autogen' || true) )
```

Comment:
Does this now match an `autogen` file located anywhere in our code base? Is that what we want? Do we need to be more / less specific?